otio_openassetio/operations/openassetio_media_linker.py

```diff
@@ -182,17 +171,26 @@ def _createSessionState(args: dict) -> SessionState:
 
     host = OTIOHostInterface()
     logger = log.SeverityFilter(log.ConsoleLogger())
-    factory = PluginSystemManagerFactory(logger)
+    factory = PythonPluginSystemManagerImplementationFactory(logger)
+
+    # check if raises
+    manager = ManagerFactory.createManagerForInterface(
+        args["identifier"], host, factory, logger
+    )
+    manager.initialize(args["settings"])
 
-    session = Session(host, logger, factory)
-    session.useManager(args["identifier"], args.get("settings", {}))
+    if not manager:
+        raise RuntimeError(
+            "No default manager configured, "
+            f"check ${ManagerFactory.kDefaultManagerConfigEnvVarName}"
+        )
```

Comment:
[fix] This check seems somewhat redundant, as we've already attempted to call `initialize` on the object so that would have raised a `RuntimeError` if `manager` was `None`.The message needs a tweak too - as it is only applicable if we called `ManagerFactory.defaultManagerForInterface`. I think `createManagerForInterface` will already have raised if there was a problem instantiating a manager with the requested identifier.

Reply:
[49c5e57](https://github.com/OpenAssetIO/otio-openassetio/pull/17/commits/49c5e5740f836b4841ffe01c17618374a0c1bfdb)

Added a test case to assert that `createManagerForInterface` does indeed throw.